Mu2eUtilities/inc/MueXSpectrum.hh

```diff
@@ -0,0 +1,58 @@
+#ifndef Mu2eUtilities_MueXSpectrum_hh
+#define Mu2eUtilities_MueXSpectrum_hh
+
+// CLHEP includes
+#include "CLHEP/Vector/LorentzVector.h"
+
+// C++ includes
+#include <utility>
+
+namespace mu2e {
+
+  double f(double E, void *p);
+
+  class MueXSpectrum {
+
+  public:
+    
+    struct Params_t {
+      double eMax;
+      double mmu;
+      double Emu;
+      double BR;
+      double Gamma;
+      double me;
+      double mN;
+      double a0;
+      double a1;
+      double a2;
+      double a3;
+      double a4;
+      double a5;
+    } _par;
+    
+    MueXSpectrum(double maxEnergy, double bin, int RadCorrected = 0);
+    
+    ~MueXSpectrum(){}
+ 
+    double getWeight                     (double E) const;
```

Comment:
I would fix the formatting (text editor is likely responsible for this....)

Reply:
what's wrong with the formatting?

Reply:
ah, ok, the spaces, removed now